watertap/unit_models/tests/test_anaerobic_digestor.py

```diff
@@ -194,13 +194,14 @@ def test_build(self, adm):
         assert hasattr(adm.fs.unit.vapor_outlet, "pressure")
 
         assert hasattr(adm.fs.unit, "ad_performance_eqn")
+        assert hasattr(adm.fs.unit, "hydraulic_retention_time")
```

Comment:
Can you add a check for HRT as well and/or volume?

Reply:
added the tests to the CSTR file, already were done in this one

Reply:
Sorry, I meant to check the value with a regression test,

Reply:
I added the tests